test/index.js

```diff
@@ -172,15 +172,15 @@ test('TypeClass', function() {
   //  Bar :: TypeClass
   var Bar = Z.TypeClass('my-package/Bar', [Foo], hasMethod('bar'));
 
-  eq(Foo['@@type'], 'sanctuary-type-classes/TypeClass');
+  eq(Foo.constructor['@@type'], 'sanctuary-type-classes/TypeClass');
```

Comment:
This pull request now uses the `type` function exported by [sanctuary-type-identifiers][1]. :)

:zap:


[1]: https://github.com/sanctuary-js/sanctuary-type-identifiers